### ISSUE TITLE
Curses not appearing on raid frames

### ISSUE THREAD

**Issue by Kosiee** [jordonwow/bigdebuffs#534]
I noticed that on my resto shaman when I spec into decurse, curses will not appear on the raid frames. I then noticed they sometimes appear as a small purple start icon on the top right of the raid frame. Is there anyway to fix this and have them appear on the raid frames like all other disppelable buffs? Thanks

**Comment by neuvo123** [comment]
untick the option on raid frames called "hide other debuffs"

i think this had been enabled trough some update but after unticking it mine works like it used to show default conditions too raid frames

**Comment by Kosiee** [comment]
Thanks, that seems to have fixed it for now.

Now to just work out how to remove the square around the unitframes and have it circle again

**Comment by svenzj** [comment]
its not fixed btw. on previous versions we would get blizzard default debuffs + bigdebuffs. nowadays it just overwrites and size etc is wrong

**Comment by Kosiee** [comment]
Yeah you are right. After testing the above mentioned fix works but also displays all the other non dissapellable buffs which I don't want to see to avoid all the buff clutter. In previous versions you would see all disspelable buffs with no issues like you said
